build.sbt

```diff
@@ -38,8 +38,8 @@ lazy val scalaHttp = crossProject.in(file("."))
     ),
     pomIncludeRepository := { _ => false },
 
-    libraryDependencies += "com.lihaoyi" %%% "utest" % "0.4.4",
-    libraryDependencies += "io.monix" %%% "monix" % "2.0.6",
+    libraryDependencies += "com.lihaoyi" %%% "utest" % "0.4.5" % Test,
```

Comment:
What is the purpose of `% Test`? Also in the README for utest, this line ends with ` % "test"`, could you explain why you used a different format?